runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/logging/DataflowWorkerLoggingInitializer.java

```diff
@@ -35,9 +35,12 @@
 import java.util.logging.LogManager;
 import java.util.logging.Logger;
 import org.apache.beam.runners.dataflow.options.DataflowWorkerLoggingOptions;
+import org.apache.beam.sdk.options.SdkHarnessOptions;
```

Comment:
Although legacy runner does not involve SdkHarnessOptions, this option is used here. It may sounds weird. This was discussed before: https://github.com/apache/beam/pull/23635#discussion_r998897966

but we chose to keep the current naming to avoid refactoring that would affecting users.